FirebaseMessaging/Apps/SwiftUISample/SwiftUISample/SwiftUISampleApp.swift

```diff
@@ -35,17 +42,61 @@ class AppDelegate: NSObject, UIApplicationDelegate, UNUserNotificationCenterDele
       }
     }
     application.registerForRemoteNotifications()
+
+    // Observe token refresh - two ways
```

Comment:
Is there a third way with async/await?

Reply:
Not for token refresh listener. It is for the async getToken call.